cpe_workspaceone/README.md

```diff
@@ -31,9 +32,15 @@ Attributes
 * node['cpe_workspaceone']['pkg']['pkg_url']
 * node['cpe_workspaceone']['pkg']['receipt']
 * node['cpe_workspaceone']['pkg']['version']
+* node['cpe_workspaceone']['pkg']['headers']
 * node['cpe_workspaceone']['prefs']
 * node['cpe_workspaceone']['uninstall']
 * node['cpe_workspaceone']['use_cache']
+* node['cpe_workspaceone']['cli_prefs']
```

Comment:
`force_list` is missing here and the documentation regarding it's use.

Reply:
Documented forcelist!